lib/core/text/abstract_text.nit

```diff
@@ -1095,6 +1095,13 @@ abstract class Text
 		end
 	end
 
+	# Copies the content of `self` to `dst`
+	#
+	# Easier, more user-friendly version of `copy_to_native`
+	#
+	# NOTE: `dst` must be large enough to withold `self.bytelen` bytes
+	fun copy_to(dst: NativeString) do copy_to_native(dst, bytelen, 0, 0)
```

Comment:
It was mainly to mimic the copy API from NativeString/NativeArray, I can change it, but I don't have a better idea for a name than this